### PR TITLE
#RI-4289 - Hide recommendations

### PR DESCRIPTION
**Pull request by zalenskiSofteq**
#RI-4289 - Hide recommendations

### CODE REVIEW

File: redisinsight/ui/src/components/live-time-recommendations/LiveTimeRecommendations.tsx [hunk 1]
```diff
@@ -103,24 +109,41 @@ const LiveTimeRecommendations = () => {
...
     isCloseEventSent.current = true
   }
 
+  const onChangeShowHidden = (value: boolean) => {
+    dispatch(setRecommendationsShowHidden(value))
+
+    sendEventTelemetry({
+      event: TelemetryEvent.INSIGHTS_RECOMMENDATION_SHOW_HIDDEN,
+      eventData: {
+        action: !value ? 'hide' : 'unhide',
```
Comment (AmirAllayarovSofteq) [inline redisinsight/ui/src/components/live-time-recommendations/LiveTimeRecommendations.tsx:124]: action should be hide/show, also i guess you missed property "provider"

File: redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx [hunk 2]
```diff
@@ -96,6 +99,24 @@ const Recommendation = ({
...
+    event.stopPropagation()
+    dispatch(
+      updateLiveRecommendation(
+        id,
+        { hide: !hide },
+        (_, { hide, name }) => sendEventTelemetry({
+          event: TelemetryEvent.INSIGHTS_RECOMMENDATION_HIDE,
+          eventData: {
+            databaseId: instanceId,
+            action: hide ? 'hide' : 'unhide',
```
Comment (AmirAllayarovSofteq) [inline redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx:112]: Same show/hide

File: redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx [hunk 3]
```diff
@@ -96,6 +99,24 @@ const Recommendation = ({
...
+            name,
```
Comment (AmirAllayarovSofteq) [inline redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx:113]: Same telemetryEvent instead name. (I guess i can add additional function to replace name with telemetryEvent or you can do it)

**Review by AmirAllayarovSofteq — CHANGES_REQUESTED** [review]
Looks good. I have only couple comments

File: redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx (hunk 2 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx]: changed, thanks

File: redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx (hunk 3 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/components/live-time-recommendations/components/recommendation/Recommendation.tsx]: changed
good catch!

**Review by AmirAllayarovSofteq — APPROVED** [review]
LGTM!

**Pull request merged**